e2e/test/scenarios/collections/instance-analytics.cy.spec.js

```diff
@@ -142,5 +144,86 @@ describeEE("scenarios > Instance Analytics Collection", () => {
         cy.contains("Move").should("not.exist");
       });
     });
+
+    it.skip("should not allow editing analytics content (#36228)", () => {
+      // dashboard
+      navigationSidebar().findByText(ANALYTICS_COLLECTION_NAME).click();
+      cy.findByTestId("pinned-items")
+        .findByText(METRICS_DASHBOARD_NAME)
+        .scrollIntoView()
+        .click();
+
+      cy.wait("@getDashboard");
+
+      cy.findByTestId("dashboard-header").within(() => {
+        cy.icon("pencil").should("not.exist");
+      });
```

Comment:
Nit:

Within is needed only for multiple assertions that are scoped to a selector. You can easily chain `.icon()` directly here.

Reply:
WDYT about adding a positive assertion before this one just to avoid false positives?